### DISCUSSION TITLE
Checking for tags in forum posts

### DISCUSSION THREAD

**Discussion by thumperkun** [discordjs/discord.js#8632] [Q&A] [answered]
### Which package is the feature request for?

discord.js

### Feature

With the new release of forum channels in Community servers:

- Is there a way to fetch the name of the forum/category that the post was created in? Seems like using `client.channels.cache.get(parentId)` does not work for forum posts (it returns `undefined`).
- Can we also fetch the tags of the newly created forum post?

### Ideal solution or implementation

Methods for fetching tags or parent category for forum channels.

**Comment by Jiralite** [comment, marked as answer]
discord.js doesn't have forum channel support yet, so the answer to both of your questions is no. You can track progress here: #7791

**Reply by thumperkun** [reply]
Cool, thanks for the update!
